### PR TITLE
Improving observability doc org, arch, and quality

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-3348

### CODE REVIEW

File: observability/main.adoc [hunk 1]
```diff
@@ -5,10 +5,10 @@ include::modules/common-attributes.adoc[]
...
 include::observability_enable.adoc[leveloffset=+2]
-include::search_intro.adoc[leveloffset=+2]
-include::manage_search.adoc[leveloffset=+2]
 include::customize_observability.adoc[leveloffset=+2]
 include::design_grafana.adoc[leveloffset=+2]
-include::insights_observability.adoc[leveloffset=+2]
+include::search_intro.adoc[leveloffset=+1]
+include::manage_search.adoc[leveloffset=+2]
+include::insights_intro.adoc[leveloffset=+1]
 include::manage_insights.adoc[leveloffset=+2]
```
Comment (dockerymick) [inline observability/main.adoc:13]: TOC reorg based on suggestions:

Observability
1. Observability service introduction (observe_environments_intro.adoc)
1.1. Observing environments (observe_environments.adoc)
1.2. Enabling observability service (observability_enable.adoc)
1.3. Customizing observability (customize_observability.adoc)
1.4. Designing your Grafana dashboard (design_grafana.adoc)
2. Searching in the console introduction (search_intro.adoc)
2.1 Managing search (manage_search.adoc)
3. Using observability with Red Hat Insights (insights_intro.adoc)
3.1. Managing insight _PolicyReports_ (manage_insights.adoc)

Comment (dockerymick) [reply observability/main.adoc:13]: FYI @berenss

**Review by oafischer — APPROVED** [review]
Looks great, thanks!

**Comment by dockerymick** [comment]
@oafischer thanks for your review!

**Pull request merged**